### DISCUSSION TITLE
farm for stars

### DISCUSSION THREAD

**Discussion by elijahpereira** [Schweinepriester/achievement-testarea#25] [General]
star for star comment when done w repo
https://github.com/elijahpereira/elijahpereira.github.io

**Comment by ArielMAJ** [comment]
🌟4🌟: https://github.com/ArielMAJ/test.me
Comment your repo here after leaving your star.

Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
https://github.com/4lfant/flutter-guidelines

**Reply by ArielMAJ** [reply]
@chriskthomas done!

**Reply by Sriansh-raj** [reply]
star my repo - https://github.com/Sriansh-raj/GitHub-Achievements

i've done yours

**Reply by michalm57** [reply]
@elijahpereira, @ArielMAJ, @4lfant, @Sriansh-raj, I marked your repo with stars! Can you do it for me, too?
https://github.com/michalm57/currency_exchange

**Reply by Sriansh-raj** [reply]
@michalm57 Done 👍🏻

**Reply by pulkitxm** [reply]
did it for you , give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by zoelesv** [reply]
Done. Here's mine, please ⭐️  https://github.com/zoelesv/Smathchat

**Reply by ArielMAJ** [reply]
Done @zoelesv

**Reply by luancesardev** [reply]
Done. Here's mine, please if you can ⭐️ https://github.com/luancesardev/project-personal-site

**Reply by fabricio-fn** [reply]
I'll leave this repository here: https://github.com/fabricio-fn/ChatGpt-API

who gives a star I return

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by ArielMAJ** [reply]
Everyone above is starred.

@4lfant I can't find your star. I've removed your star until you star me back

**Reply by michalm57** [reply]
@fabricio-fn, @luancesardev, @ArielMAJ,  done please star mine https://github.com/michalm57/currency_exchange.

**Reply by cyfyifanchen** [reply]
@fabricio-fn @luancesardev @ArielMAJ @michalm57 I stared you guys, star back please. https://github.com/cyfyifanchen/dotfiles

**Reply by fabricio-fn** [reply]
Done 🤟 @cyfyifanchen

**Reply by ArielMAJ** [reply]
> @fabricio-fn @luancesardev @ArielMAJ @michalm57 I stared you guys, star back please. https://github.com/cyfyifanchen/dotfiles
> 

Done

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ArielMAJ** [reply]
Done @Mattral

**Reply by fabricio-fn** [reply]
Done, @Mattral 

Please mine too https://github.com/fabricio-fn/ChatGpt-API

**Reply by Mattral** [reply]
Done! Thank you

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by AdityaNow** [reply]
Done 😃 (feel free to star any of my repos https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Thank you

**Reply by ganochenkodg** [reply]
@ArielMAJ @xommmax @michalm57 @Pulkitxm @zoelesv @luancesardev @fabricio-fn @cyfyifanchen @ternera @AdityaNow - done! and my pls https://github.com/ganochenkodg/qdrant-operator

**Reply by fabricio-fn** [reply]
@ganochenkodg done!

**Reply by cold12coffee-pixel** [reply]
i have starred yours please star mine https://github.com/ArshiBansal/Stock_Market_Analysis

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Reply by Rantoryu** [reply]
@4lfant  Done, please star mine too :) https://github.com/Rantoryu/Personal_Website

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by michalm57** [reply]
@fabricio-fn done, please star mine https://github.com/michalm57/currency_exchange.

**Reply by Mattral** [reply]
Done!

Please mine too
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by AdityaNow** [reply]
Done 😃 (feel free to star any of my repo https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Thank you

**Reply by cyfyifanchen** [reply]
@AdityaNow done on yours. 

Here is mine: https://github.com/cyfyifanchen/dotfiles-and-tools

**Reply by cold12coffee-pixel** [reply]
please star mine and in return i will do

https://github.com/ArshiBansal/Stock_Market_Analysis

**Comment by chriskthomas** [comment]


**Reply by Sriansh-raj** [reply]
star my repo - https://github.com/Sriansh-raj/GitHub-Achievements

I have done yours

**Reply by Rantoryu** [reply]
@chriskthomas I gave You a star :)
Please star mine too: https://github.com/Rantoryu/Personal_Website

**Reply by pulkitxm** [reply]
@Sriansh-raj @Rantoryu  give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by Rantoryu** [reply]
@Pulkitxm done :)

**Reply by Sriansh-raj** [reply]
@Pulkitxm done :)

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by Sriansh-raj** [reply]
@fabricio-fn done :)

**Reply by michalm57** [reply]
@Sriansh-raj, @Rantoryu done please star mine https://github.com/michalm57/currency_exchange.

**Reply by Sriansh-raj** [reply]
@michalm57 done :)

**Reply by Sriansh-raj** [reply]
@cyfyifanchen Done 👍🏻  Nice project though :)

**Reply by ArielMAJ** [reply]
> @Sriansh-raj done https://github.com/cyfyifanchen/dotfiles

Done

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by michalm57** [comment]
@elijahpereira I marked your repo with stars! Can you do it for me, too?
https://github.com/michalm57/currency_exchange

**Reply by pulkitxm** [reply]
did it for you , give mine as well please
https://github.com/Pulkitxm/Delta-Apna-College

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by michalm57** [reply]
@fabricio-fn done, please star mine https://github.com/michalm57/currency_exchange

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by pulkitxm** [comment]
star my  [repo](https://github.com/Pulkitxm/Delta-Apna-College), I'll do it for yours
 [https://github.com/Pulkitxm/Delta-Apna-College](https://github.com/Pulkitxm/Delta-Apna-College)
Have a nice Day !

**Reply by ArielMAJ** [reply]
@Pulkitxm I starred you, please star my repo too: https://github.com/ArielMAJ/test.me

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by michalm57** [reply]
@fabricio-fn done, please star mine https://github.com/michalm57/currency_exchange

**Reply by fabricio-fn** [reply]
Done

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by luancesardev** [comment]
https://github.com/luancesardev/project-personal-site

Star my repo and reply with your repo to get a star ⭐️ Thanks *!

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by luancesardev** [reply]
Done.

**Reply by michalm57** [reply]
@luancesardev done, please star mine:
https://github.com/michalm57/currency_exchange

**Reply by Rantoryu** [reply]
@luancesardev done, please star mine too:
https://github.com/Rantoryu/Personal_Website

**Reply by luancesardev** [reply]
Done ⭐️

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by ganochenkodg** [reply]
@elijahpereira @luancesardev done, and mine pls https://github.com/ganochenkodg/qdrant-operator

**Comment by lqq-code** [comment]
https://github.com/lqq-code/threejs-rabbit-island

Star my repo and reply with your repo to get a star ⭐️ Thanks *!

**Reply by zoelesv** [reply]
Done. That's so cool. Here's mine https://github.com/zoelesv/Smathchat

**Reply by lqq-code** [reply]
done！

**Reply by fabricio-fn** [reply]
I gave you a star, return it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by lqq-code** [reply]
done

**Reply by michalm57** [reply]
@lqq-code  done, please star mine: https://github.com/michalm57/currency_exchange

**Reply by Rantoryu** [reply]
@lqq-code I gave you a star, i would appreciate starring mine too:
https://github.com/Rantoryu/Personal_Website

**Reply by fabricio-fn** [reply]
Done

**Reply by cyfyifanchen** [reply]
@lqq-code done https://github.com/cyfyifanchen/dotfiles

**Reply by Mattral** [reply]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by fabricio-fn** [reply]
Done, @ternera 

Please mine too https://github.com/fabricio-fn/ChatGpt-API

**Reply by Mattral** [reply]
Done @ternera @fabricio-fn

**Reply by Mattral** [reply]
lets connect by following each other too!

**Reply by the1Riddle** [reply]
hello @lqq-code please look on mine too

here is the link:👉 [my repo](https://github.com/the-1Riddle/packageInstallationHub)

thanks ❤️

**Reply by ganochenkodg** [reply]
@lqq-code done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Reply by cyfyifanchen** [reply]
@ganochenkodg done. 

https://github.com/cyfyifanchen/dotfiles-and-tools

**Comment by fabricio-fn** [comment]
I'll leave this repository here: https://github.com/fabricio-fn/ChatGpt-API

who gives a star I return

**Reply by michalm57** [reply]
@fabricio-fn done, please star mine: https://github.com/michalm57/currency_exchange

**Reply by Rantoryu** [reply]
@fabricio-fn I've starred yours, can you do mine too?
https://github.com/Rantoryu/Personal_Website

**Reply by sree-hari-s** [reply]
@michalm57 @fabricio-fn @Rantoryu I have starred your's repos can you do mine too?
https://github.com/sree-hari-s/Ecommerce_New_Project

**Reply by fabricio-fn** [reply]
@sree-hari-s done!

**Reply by michalm57** [reply]
@sree-hari-s done

**Reply by ArielMAJ** [reply]
@sree-hari-s I starred you, please star my repo too: https://github.com/ArielMAJ/test.me

**Reply by sree-hari-s** [reply]
@ArielMAJ done

**Reply by dexexpert** [reply]
https://github.com/t1nyTi/TombFork
@sree-hari-s 
@ArielMAJ 
@fabricio-fn 
done, Please star my repo

**Reply by sree-hari-s** [reply]
@t1nyTi done

**Reply by fabricio-fn** [reply]
@t1nyTi Done!

**Reply by ArielMAJ** [reply]
TY @t1nyTi done!

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by ganochenkodg** [reply]
@sree-hari-s @t1nyTi done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Comment by Mast3r0mid** [comment]
https://github.com/Mast3r0mid/TaskBarDragAndDropNoUAC

I'll Leave my repo here

**Reply by fabricio-fn** [reply]
done!

please reciprocate: https://github.com/fabricio-fn/ChatGpt-API

**Reply by Mattral** [reply]
Done!

Please mine too
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by ternera** [reply]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by Mattral** [reply]
Done!

**Reply by the1Riddle** [reply]
hello @Mast3r0mid just done with it.

here is the link:👉 [my repo](https://github.com/the-1Riddle/packageInstallationHub)

thanks ❤️

**Reply by ganochenkodg** [reply]
@Mast3r0mid done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Comment by ternera** [comment]
Please star and I will return it: https://github.com/ternera/ternera.org

**Reply by fabricio-fn** [reply]
ready, reciprocate here https://github.com/fabricio-fn/ChatGpt-API

**Reply by Mattral** [reply]
Done!

Please mine too
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by fabricio-fn** [reply]
@ternera done!

ready, reciprocate here https://github.com/fabricio-fn/ChatGpt-API

**Reply by fabricio-fn** [reply]
@ternera  I already gave you Estela but you still haven't returned it here: https://github.com/fabricio-fn/ChatGpt-API

**Reply by ArielMAJ** [reply]
@ternera https://github.com/ArielMAJ/test.me

**Reply by cyfyifanchen** [reply]
@ternera done.

https://github.com/cyfyifanchen/dotfiles-and-tools

**Comment by AdityaNow** [comment]
Done 😃  please feel free to star my repo  - https://github.com/AdityaNow/NavtorAssignment

**Reply by ArielMAJ** [reply]
Done @AdityaNow https://github.com/ArielMAJ/test.me

**Reply by AdityaNow** [reply]
@ArielMAJ , Done 😃

**Comment by the1Riddle** [comment]
hello @Mast3r0mid , @lqq-code , @luancesardev , @michalm57 and @elijahpereira 

just star your repos, in exchange please star mine too...
here is the link:👉 [my repo](https://github.com/rezzcode/Epubespiar)

thanks ❤️

**Reply by ArielMAJ** [reply]
Done @the1Riddle. https://github.com/ArielMAJ/test.me

**Reply by the1Riddle** [reply]
thanks , i had stared yours too

**Reply by cyfyifanchen** [reply]
@the1Riddle  done. https://github.com/cyfyifanchen/dotfiles-and-tools

**Reply by the1Riddle** [reply]
@cyfyifanchen thanks for taking the time to star my repo, I've also done that in exchange.

**Comment by giangeralcus** [comment]
Starred everyone above in this thread ⭐ — star for star please: https://github.com/giangeralcus/gg-tetris (comment here after starring and I'll return it!)

**Comment by giangeralcus** [comment]
Still exchanging — I've starred the repos from the participants above (verified via their /stargazers before commenting). My repos if anyone wants to swap: giangeralcus/gg-tetris, giangeralcus/badge-lab. Star any and comment here, I'll star back the same day.
